src/main/kotlin/no/fdk/dataset_catalog/configuration/SecurityProperties.kt

```diff
@@ -0,0 +1,10 @@
+package no.fdk.dataset_catalog.configuration
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties("security")
+data class SecurityProperties(
```

Comment:
Den virker unødvendig. Issuer er jo en standard del av JWT verifisering

Reply:
Ja, men vi overskriver standard verifisering for å få med audience

Reply:
Issuer og audience er forskjellige. Issuer skal være en standard egenskap. Property kan legges til (se kommentar nede)

Reply:
Er ikke også expiry time en standard egenskap?